riscv/plic.cc

```diff
@@ -313,8 +316,16 @@ bool plic_t::load(reg_t addr, size_t len, uint8_t* bytes)
       return false;
   }
 
-  if (PRIORITY_BASE <= addr && addr < ENABLE_BASE) {
+  if (PRIORITY_BASE <= addr && addr < PENDING_BASE) {
     ret = priority_read(addr, &val);
+  } else if (PENDING_BASE <= addr && addr < ENABLE_BASE) {
+    uint32_t id_word = (addr - PENDING_BASE) >> 2;
```

Comment:
To be consistent with other parts of PLIC emulation, add `bool plic_t::priority_read(reg_t offset, uint32_t *val)` and use it here.

Reply:
I assume you meant `pending_read()`?

Reply:
Ahh, yes. It was a typo. I have fixed it.